### PR TITLE
Upgrade to angular 15

### PR DESCRIPTION
**Pull request by OscarDeDios**
# PR Details

Update to Angular 15 and TypeScript 4.8

## Description

Followed Angular Update Guide

https://update.angular.io/?l=3&v=14.0-15.0

## Related Issue
#672

## Motivation and Context

We want to have the latest technology.

## How has this been tested

It has been executed tests and tested in a real project with npm link.

## Types of changes

- [x] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] I have read the CONTRIBUTING document
- [x] My code follows the code style of this project
- [ ] My change requires a change to the documentation

### CODE REVIEW

**Review by fjaguado — APPROVED** [review]

**Review by RubKrmon — DISMISSED** [review]

**Review by alfonsserra — APPROVED** [review]
Great work @OscarDeDios

File: package.json [hunk 1]
```diff
@@ -15,24 +15,21 @@
...
+    "@angular/platform-browser": "15.0.4",
+    "@angular/platform-browser-dynamic": "15.0.4",
+    "@angular/router": "15.0.3",
+    "@ngx-translate/core": "14.0.0",
+    "@ngx-translate/http-loader": "^7.0.0",
     "ag-grid-angular": "^28.2.1",
     "ag-grid-community": "^28.2.1",
     "angular-split": "^4.0.0",
     "bootstrap": "^4.5.2",
-    "chart.js": "^2.9.3",
```
Comment (fjaguado) [inline package.json:35]: Is not a peerDependency of another systelab library??

**Comment by fjaguado** [comment]
In general the are a lot of code format changes and I don't know if we have eslint correctly configured in our IDEs

**Review by RubKrmon — APPROVED** [review]

**Review by carlosra85 — COMMENTED** [review]
Good job! Thanks!!

**Pull request merged**